scrapers/peeonher.yml

```diff
@@ -0,0 +1,28 @@
+name: peeonher
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - peeonher.com/updates/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //h1[@class='page_title']
+     Date:
+        selector: //strong[text()="Published"]/following-sibling::text()
+        postProcess:
+          - replace:
+            - regex: (\d+)[dhnrst]+
+              with: $1
+          - parseDate: 2 January 2006
+     Performers:
+      Name: //div[@class="pull-left"]//a/text()
```

Comment:
this doesnt seem to work always eg `https://www.peeonher.com/updates/pissing-foursome/`
```yaml
      Performers:
        Name: //a[@class="sub_sub_model"]
```
should be enough

Reply:
Yup good spotting.